workflows/diagnostics/fv3net/diagnostics/offline/compute.py

```diff
@@ -171,10 +172,11 @@ def _compute_diagnostics(
 
     # then average over the batches for each output
     ds_summary = xr.concat(batches_summary, dim="batch")
+    ds_summary = insert_r2(ds_summary)
+    if DATASET_DIM_NAME in ds_summary.dims:
```

Comment:
I think the handling of the aggregate-over-dataset-dim quantities could be consolidated into the if statement  if  1) `insert_r2` is removed and 2) a helper `insert_aggregate_r2` similar to `insert_aggregate_bias` calls `_compute_r2` on `ds_summary.mean("dataset")` and uses `rename_via_replace` in a similar manner. Just suggesting since it was clear from reading the code that the aggregated bias was being calculated and inserted, but it was not obvious that the same was true for the R2 unless looking into the `insert_r2` code.

e.g. 
```
ds_summary.update( _compute_r2(ds_summary, tag="_r2")) )
if DATASET_DIM_NAME in ds_summary.dims:
    ds_summary = insert_aggregate_r2(ds_summary)
    ds_summary = insert_aggregate_bias(ds_summary)    
```

Reply:
Indeed that's much clearer, and further simplifies the code in `_helpers.py`.